Automattic/woocommerce-payments
# UPE - Fix saving payment methods on WC Blocks if a non-reusable method is enabled

Fixes #3670 

#### Changes proposed in this Pull Request

Saving UPE payment methods on WooCommerce Blocks breaks when a non-reusable method like Giropay is enabled. While doing checkout, the chosen payment method is not passed along causing problem with updating the payment intent, triggering the error .

#### Testing instructions

* Follow instruction on #3670 to replicate the problem
* Checkout the branch `fix/blocks-upe-save-payment-method` & build assets
* Follow the instructions again and confirm that checkout is successful & payment method is saved as expected.

-------------------


**Post merge**
